test/drivers/test_cqrrp.cc

```diff
@@ -227,49 +290,103 @@ TEST_F(TestCQRRP, CQRRP_blocked_low_rank) {
 #endif
 }
 
+// Note: If Subprocess killed exception -> reload vscode
+TEST_F(TestCQRRP, CQRRP_blocked_near_zero_input_qp3) {
+    int64_t m = 1000;//5000;
+    int64_t n = 1000;//2000;
+    int64_t k = 1000;
+    double d_factor = 1;//1.0;
+    int64_t b_sz = 100;//500;
+    double norm_A = 0;
+    double tol = std::pow(std::numeric_limits<double>::epsilon(), 0.85);
+    auto state = RandBLAS::RNGState();
 
+    CQRRPTestData<double> all_data(m, n, k);
+    RandLAPACK::CQRRP_blocked<double, r123::Philox4x32> CQRRP_blocked(true, tol, b_sz);
+    CQRRP_blocked.nnz = 2;
+    CQRRP_blocked.num_threads = 4;
+    CQRRP_blocked.use_qp3 = true;
+
+    std::fill(&(all_data.A.data())[0], &(all_data.A.data())[m * n], 0.0);
+    all_data.A[1000*200 + 10] = 1;
+
+    norm_and_copy_computational_helper(norm_A, all_data);
+#if !defined(__APPLE__)
+    test_CQRRP_general(d_factor, norm_A, all_data, CQRRP_blocked, state);
+#endif
```

Comment:
This should be much more prominent. Wrap the entire test fixture.

Reply:
Done.